client/context.go

```diff
@@ -387,3 +388,9 @@ func NewKeyringFromBackend(ctx Context, backend string) (keyring.Keyring, error)
 
 	return keyring.New(sdk.KeyringServiceName(), backend, ctx.KeyringDir, ctx.Input, ctx.KeyringOptions...)
 }
+
+func (ctx Context) WithAliases(aliases map[string]string) Context {
```

Comment:
WithAliases is too generic. Consider `WithIBCAliases`.

Reply:
Should be denom aliases, applies equally to tokenfactory